charts/gateway/templates/node-properties-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if not .Values.disklessConfig  }}
```

Comment:
See https://github.com/CAAPIM/apim-charts/blob/node-properties/charts/gateway/templates/license-secret.yaml. We need to add the same condition otherwise node.properties will get created over and over again.

maybe we need to make disklessConfig a section in values.yaml instead like:
```
disklessConfig:
  enabled: false
  existingNodePropertiesName: myNodeProperties
```

For nodePropertiesSecretName to get a value, need to the same thing done for gateway.license here:
https://github.com/CAAPIM/apim-charts/blob/node-properties/charts/gateway/templates/_helpers.tpl#L188

Reply:
Actual volume mount is also missing. License example: https://github.com/CAAPIM/apim-charts/blob/node-properties/charts/gateway/templates/deployment.yaml#L189

Reply:
I changed it 

```
disklessConfig:
  enabled: false
  value:
  # existingSecretName:
```